lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/FunctionalChaosResult.hxx

```diff
@@ -134,6 +146,12 @@ private:
   /** Composed meta model */
   Function composedMetaModel_;
 
+  /** Is regression? */
```

Comment:
Are these attributes really mandatory? You are checking properties, the value could be hard coded into the many isModelSelection() and isLeastSquares() methods.

Reply:
The attributes are necessary in FCResult, because we cannot get that information after the object is created. However, I updated the code of all the methods that implements these services e.g. in AdaptiveStrategy, PenalizedLeastSquaresAlgorithm, LeastSquaresMetaModelSelection, CleaningStrategy. This is done in 8d8ebe32519c39c30f3989424581f6336673145e. 

Here is an example in [CleaningStrategy.cxx#L289](https://github.com/mbaudin47/openturns/blob/d2d18e244254b79129c880b1857a7c416ff819f3/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/CleaningStrategy.cxx#L289) and another in [IntegrationStrategy.cxx#L212](https://github.com/mbaudin47/openturns/blob/d2d18e244254b79129c880b1857a7c416ff819f3/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/IntegrationStrategy.cxx#L212).